src/page-building/career/openings.tsx

```diff
@@ -33,7 +33,7 @@ export const Openings = (props: OpeningsProps) => {
       <SectionHeadline>Unsere offenen Stellen</SectionHeadline>
       <TeaserGrid>
         {props.jobs.map((item) => (
-          <Teaser title={item.name} linkTo={item.fields.path} key={item.id}>
+          <Teaser title={item.name} linkTo={item.fields?.path} key={item.id}>
```

Comment:
This fails during runtime as this is a special key created by gatsby graphql and not provided in the example data set